### PR TITLE
feat: implement wallet transaction export modal ux

### PR DESCRIPTION
**Pull request by goga-m**
## Checklist

-   [x] My changes look good in both light AND dark mode
-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

File: src/domains/transaction/components/TransactionExportModal/TransactionExportForm/CsvSettings.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	return (
+		<FormField name="delimiter">
+			<Dropdown
+				data-testid="TransactionExportForm--delimiter-options"
+				options={delimiterOptions.options}
+				onSelect={(option) => onSelect?.(option.value as CSVDelimiter)}
+				toggleContent={(isOpen: boolean) => (
+					<CollapseToggleButton
+						isOpen={isOpen}
+						className="w-full cursor-pointer justify-between space-x-4 overflow-hidden rounded-xl border border-theme-success-100 p-3 dark:border-theme-secondary-800 sm:p-6 md:w-auto md:space-x-2 md:rounded md:border-0 md:border-none md:py-2 md:px-0"
```
Comment (dated) [inline src/domains/transaction/components/TransactionExportModal/TransactionExportForm/CsvSettings.tsx:33]:
```suggestion
						className="w-full cursor-pointer justify-between space-x-4 overflow-hidden rounded-xl border border-theme-success-100 p-3 dark:border-theme-secondary-800 sm:p-6 md:w-auto md:space-x-2 md:rounded md:border-0 md:border-none md:py-0 md:px-0"
```

File: src/domains/transaction/components/TransactionExportModal/TransactionExportForm/BasicSettings.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	const { isXs } = useBreakpoint();
+
+	return (
+		<ListDivided
+			items={[
+				{
+					content: isXs && <TranctionTypeOptions />,
+					label: t("TRANSACTION.EXPORT.FORM.TRANSACTIONS"),
+					value: !isXs && <TranctionTypeOptions />,
+					wrapperClass: "py-4",
```
Comment (dated) [inline src/domains/transaction/components/TransactionExportModal/TransactionExportForm/BasicSettings.tsx:83]:
```suggestion
					wrapperClass: "pb-4",
```

**Review by dated — COMMENTED** [review]
some minor padding adjustments

**Comment by goga-m** [comment]
Closing in favor of https://github.com/ArdentHQ/arkvault/pull/71

**Pull request closed without merging**